packages/concrete5_theme/themes/concrete5/css/build/stripes.less

```diff
@@ -658,7 +658,7 @@ p.c5-right-balance {
   display: block !important;
   white-space: normal !important;
   font-size: 16px !important;
-
+  margin-bottom: 50px !important;
```

Comment:
Is this going to impact existing styles on the existing marketing sites?

Reply:
good question, I'm going to use the helper class instead.